### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by loboulet**
We update dependencies to stay up-to-date 🎉

### CODE REVIEW

**Comment by charlesdemers** [comment]
We should add an entry in the changelog

File: package.json [hunk 1]
```diff
@@ -39,34 +39,33 @@
...
+    "ember-cli-htmlbars": "4.2.0",
+    "ember-cli-inject-live-reload": "2.0.2",
     "ember-cli-sass": "10.0.1",
     "ember-cli-sri": "2.1.1",
-    "ember-cli-typescript": "3.0.0",
+    "ember-cli-typescript": "3.1.2",
     "ember-cli-uglify": "3.0.0",
-    "ember-concurrency": "1.1.0",
+    "ember-concurrency": "1.1.3",
     "ember-concurrency-decorators": "machty/ember-concurrency-decorators#dc550f22d94a1ca056be8e8022f8c294ab4e6c87",
```
Comment (charlesdemers) [inline package.json]: You can bump this to `1.0.0`, we tested it in a project and it works well!

**Comment by loboulet** [comment]
> We should add an entry in the changelog

@charlesdemers I was not sure about this one since almost all packages have been updated. Do I really need to list them all 😬 ?

**Comment by charlesdemers** [comment]
Nope, just write that a bunch of dependencies were updated

**Pull request merged**